srcpkgs/inih/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'inih'
+pkgname=inih
+version=r48
+revision=1
+archs="noarch"
```

Comment:
`noarch` is for things like python packages that are not compiled for specific architectures.